spec/views/users/user_index_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'users#index', type: :feature do
+  before(:each) do
+    @first_user = User.create(name: 'Kim', photo: 'https://picsum.photos/200/300', bio: 'I am a Software Engineer',
+                              postsCounter: 2)
+    @second_user = User.create(name: 'Sarah', photo: 'https://picsum.photos/200/300', bio: 'I am a teacher',
+                               postsCounter: 3)
+
+    visit '/'
+  end
+
+  it 'can see the username of all other users.' do
+    expect(page).to have_content(@first_user.name)
+    expect(page).to have_content(@second_user.name)
+  end
+
+  it 'can see the profile picture for each user.' do
+    expect(page).to have_css("img[src*='https://picsum.photos/200/300']")
+  end
+
+  it 'can see the number of posts each user has written.' do
+    expect(page).to have_content(@first_user.postsCounter)
+    expect(page).to have_content(@second_user.postsCounter)
+  end
+
+  it 'When I click on a user, I am redirected to that users show page.' do
+    click_link(@first_user.name)
+    expect(page).to have_content(@first_user.name)
+    expect(page).to have_content(@first_user.bio)
+    expect(page).to have_current_path(user_path(@first_user))
+  end
```

Comment:
- [ ] You have fixed the majority of the tests 👏 however, the tests pertaining to the `user_index` are failing as you can see below 👇 Please try to make them green ✔️ 
![image](https://user-images.githubusercontent.com/46839548/227669908-0b7ab7e4-1258-464b-8877-6276169eebc6.png)

Reply:
Hi @codepantha 
can you please attach a screenshot of the error messages? because I'm not seeing the errors on my end, all the tests are passed

![test](https://user-images.githubusercontent.com/70941894/227688992-93180104-48fe-41fe-a057-c2b057402e3c.PNG)

![test1](https://user-images.githubusercontent.com/70941894/227688988-7013047d-6a0b-4070-801b-d5082082646b.PNG)